demos/projects/ST/b-l475e-iot01a/GSG.md

```diff
@@ -267,4 +262,4 @@ To remove the entire Azure IoT Central sample application and all its devices an
 
 In this tutorial you built a custom image that contains Azure FreeRTOS middleware sample code, and then flashed the image to the STM DevKit device. You also used the IoT Central portal to create Azure resources, connect the STM DevKit securely to Azure, view telemetry, and send messages.
 
-* For device developers, the suggested next step is to see the other tutorials in the series [Getting started with Azure RTOS](https://go.microsoft.com/fwlink/p/?linkid=2129824).
+* For device developers, the suggested next step is to see the other tutorials in the series [Getting started with FreeRTOS]().
```

Comment:
Is this missing a link?

Reply:
removed the link, it will be added when the link is ready from the docs page (not available yet)